firefox-ios/Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -885,6 +887,22 @@ extension HomepageViewController: UIPopoverPresentationControllerDelegate {
     }
 }
 
+// MARK: - UIContextMenuInteractionDelegate
+extension HomepageViewController: UIContextMenuInteractionDelegate {
+    // Handles iPad trackpad right clicks
+    func contextMenuInteraction(_ interaction: UIContextMenuInteraction,
+                                configurationForMenuAtLocation location: CGPoint)
+    -> UIContextMenuConfiguration? {
```

Comment:
```suggestion
    func contextMenuInteraction(
        _ interaction: UIContextMenuInteraction,
        configurationForMenuAtLocation location: CGPoint
    ) -> UIContextMenuConfiguration? {
```

That formatting was weird for swift. lol. I'd recommend letting Xcode do the formatting for this kiinda stuff by just hitting Control + M (or Editor > Structure > Format to Multiple Lines) and it will do it nicely.